core/federated/RTI/rti_remote.h

```diff
@@ -22,6 +22,7 @@
 #include <unistd.h>     // Defines read(), write(), and close()
 #include <strings.h>    // Defines bzero().
 
+#include "net_util.h"
```

Comment:
Why is this added?

Reply:
It was needed because the socket_type_t was moved to socket_common.h, however we do not need this anymore. I'll remove this.